### PR TITLE
Update representation of print version

### PR DESCRIPTION
**Pull request by Ge11ert**

### CODE REVIEW

File: src/app/components/sidebar/sidebar.jsx [hunk 1]
```diff
@@ -147,17 +161,19 @@ class Sidebar extends React.Component {
...
-          maxSize={{ width: maxWidth }}
-          onResize={this.onResize}
-          onResizeStop={this.onResizeStop}
-        >
-          {children}
-        </Resizable>
+        { resizableEnabled ? (
+          <Resizable
+            ref={this.resizable}
+            mix={b('sidebar__resizable')}
```
Comment (igoradamenko) [inline src/app/components/sidebar/sidebar.jsx:167]: Do we need `b` here? 🤔

Comment (Ge11ert) [reply src/app/components/sidebar/sidebar.jsx:167]: In some other internal projects we have an agreement to use b whether mods/mix are applied or not. This brings kind of standardization and we are (I am) used to write like this

Comment (igoradamenko) [reply src/app/components/sidebar/sidebar.jsx:167]: Huh. Okay.

**Comment by bivihoba** [comment]
LGTM :-)

**Comment by igoradamenko** [comment]
Published @funboxteam/blueprinter@5.2.0.

Thanks! 🐨

**Pull request merged**